content/applications/finance/expenses/log_expenses.rst

```diff
@@ -0,0 +1,240 @@
+- :guilabel:`Digitize on demand only`: this option only digitizes receipts when requested. A
+  :guilabel:`Digitize document` button appears on expense records. When clicked, the receipt is
+  scanned and the expense record is updated.
+- :guilabel:`Digitize automatically`: this option automatically digitizes all receipts when they are
+  uploaded.
+
+Beneath these options are two additional links. Click the :icon:`fa-arrow-right` :guilabel:`Buy
+credits` button to purchase credits for receipt digitization. Click the :icon:`fa-arrow-right`
+:guilabel:`View My Services` button to view a list of all current services, and their remaining
+credit balances.
+
+For more information on document digitization and :abbr:`IAP's (in-app purchases)`, refer to the
+:doc:`In-app purchase (IAP) <../../essentials/in_app_purchase>` documentation.
+
+.. note::
+   The :guilabel:`Do not digitize` option for :guilabel:`Expense Digitization (OCR)` may seem like a
+   redundancy. It seems to be the same as disabling the :guilabel:`Expense Digitization (OCR)`
+   option.
+
+   When the :guilabel:`Expense Digitization (OCR)` option is enabled, a module is installed so that
+   receipts can be scanned. Disabling this option would uninstall the module.
+
+   If at some point, there is a desire to temporarily stop digitizing receipts, select the
+   :guilabel:`Do not digitize` option. The reason this option is available is so the module is not
+   uninstalled, allowing for digitization to be enabled in the future by selecting one of the other
+   two options.
+
+Upload receipts
+---------------
+
+First, access the main :guilabel:`Expenses` app dashboard view by navigating to
+:menuselection:`Expenses app --> My Expenses --> My Expenses`. Click :guilabel:`Upload`, and a file
+explorer appears. Navigate to the desired receipt, click on it to select it, then click
+:guilabel:`Open`.
+
+.. image:: log_expenses/upload.png
+   :align: center
+   :alt: Create an expense by scanning a receipt. Click Scan at the top of the Expenses dashboard
+         view.
+
+The receipt is scanned, and a new expense record is created. The :guilabel:`Expense Date` field is
+populated with today's date, along with any other fields based on the scanned data, such as the
+:guilabel:`Total`.
+
+Click on the new entry to open the individual expense form, and make any changes needed. The scanned
+receipt appears in the chatter.
+
+.. _expenses/email_expense:
+
+Email expenses
+==============
+
+Instead of individually creating each expense in the *Expenses* app, expenses can be automatically
+created by sending an email to an email alias.
+
+To do so, an email alias must first be configured. Navigate to :menuselection:`Expenses app -->
+Configuration --> Settings`. Ensure :guilabel:`Incoming Emails` is enabled. The default email alias
+is :guilabel:`expense@(domain).com`. Change the email alias by entering the desired email in the
```

Comment:
```suggestion
is *expense@(domain).com*. Change the email alias by entering the desired email in the
```

Reply:
I used a GUI because it does automatically populate and you see it on the UI- but I do like thee italics better.